src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -1219,6 +1252,65 @@ internal async void OnTimer(object sender, System.Timers.ElapsedEventArgs e)
             }
         }
 
+        internal void OnInvocationTimer(object sender, System.Timers.ElapsedEventArgs e)
+        {
+            if (_disposed)
+            {
+                return;
+            }
+
+            try
+            {
+                CheckAndRecycleFaultyLanguageWorker();
+            }
+            catch
+            {
```

Comment:
Shouldn't we log the Exception details here?

Reply:
To add, any time we're doing a catch all like this, we need to be using the IsFatal extension method to ensure we're not trapping OOM, etc. level exceptions. You can see other examples in the code.

Reply:
Updated code here. Please check if this looks correct.